app/models/mybookings/booking.rb

```diff
@@ -31,6 +31,7 @@ def self.by_start_date_group_by_resource_type
     def self.new_for_user user, params, event_type
       booking = self.new(params)
       booking.user = user
+      booking.recurrent_type = 'daily' if booking.is_a_no_repeat_booking?
```

Comment:
Why do we set it to daily? Why can't be blank if it comes as a blank from the form?

Reply:
I don't leave it blank because in other parts of the code we make a dynamic call using the value of the resource type variable. I could actually leave it blank but I could mean more changes to the code than this simple line. I will leave a comment to clarify this.